From 6937257b0c4acb16a62ed04cfbeb20a44cf22fd8 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 31 Jan 2013 15:02:51 +0000
Subject: [PATCH 0394/1077] LogService.exe V1.0.4.4 massaiSNMP.dll V1.0.4.4
 Mantis 0025566: WeightScale: support standalone Scale (BaggageScale CUSS1.3)
 new event-/comp-configuration for WeightScaleDevice and
 massai.notification.drivers.ScaleDev.Scale

git-svn-id: svn://localhost/SelfServiceCommon/trunk@475 90b65887-3827-0410-9a23-83215b262276
---
 .../WEIGHT_SCALE_DEVICE.GENERAL.compDefinition.csp     |  1 +
 .../compSource/WEIGHT_SCALE_DEVICE.compDefinition.cfg  |  3 +++
 .../Massai/cpp/LogService/eventConfiguration.txt       | 10 ++++++++++
 .../Massai/cpp/LogService/inc/LogService_def.h         | 10 +++++++---
 .../Massai/cpp/LogService/inc/massaiSNMP_def.h         | 10 +++++++---
 5 files changed, 28 insertions(+), 6 deletions(-)
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/WEIGHT_SCALE_DEVICE.GENERAL.compDefinition.csp
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/WEIGHT_SCALE_DEVICE.compDefinition.cfg

diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/WEIGHT_SCALE_DEVICE.GENERAL.compDefinition.csp b/SelfServiceCommon/Massai/cpp/LogService/compSource/WEIGHT_SCALE_DEVICE.GENERAL.compDefinition.csp
new file mode 100644
index 00000000..3f06aed5
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/WEIGHT_SCALE_DEVICE.GENERAL.compDefinition.csp
@@ -0,0 +1 @@
+HARDWARE_OK;&;SCALE_COMP.OK;WEIGHT_SCALE_DEVICE.GENERAL.compDefinition
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/WEIGHT_SCALE_DEVICE.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/WEIGHT_SCALE_DEVICE.compDefinition.cfg
new file mode 100644
index 00000000..cb02b19b
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/WEIGHT_SCALE_DEVICE.compDefinition.cfg
@@ -0,0 +1,3 @@
+SCALE_INPUT;OK=2070;ERROR=2072 | 2073 | 2074 | 2075
+SCALE_COMP;OK=SCALE_INPUT.OK
+CUSS_SCALE_COMP(HIDDEN);OK=SCALE_COMP.OK
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt b/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
index 26db1ec9..cf9084d2 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
+++ b/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
@@ -89,6 +89,12 @@
 2065;30305,ATB_ESCR_CAPTURE;[STAT][LOG][SNMP:#:1:30];Cardcapture unavailable: NOT_RESPONDING
 2066;30106,ATB_ESCR_CAPTURE;[STAT][LOG][SNMP:#:1:30];Cardcapture unavailable: MEDIA_FULL
 
+2070;10000,WeightScaleDevice;[STAT][LOG][SNMP:#:1:30];Scale available
+2072;30302,WeightScaleDevice;[STAT][LOG][SNMP:#:1:30];Scale unavailable: HARDWARE_ERROR
+2073;30303,WeightScaleDevice;[STAT][LOG][SNMP:#:1:30];Scale unavailable: CRITICAL_SOFTWARE_ERROR
+2074;30304,WeightScaleDevice;[STAT][LOG][SNMP:#:1:30];Scale unavailable: NOT_REACHABLE
+2075;30305,WeightScaleDevice;[STAT][LOG][SNMP:#:1:30];Scale unavailable: NOT_RESPONDING
+
 // old feeder definition >>>> -------------------------------------------------------------
 2100;10000,ATB2Bin1;[STAT][LOG][SNMP:#:1:30];ATB - Printer Stock 1 available
 2101;10000,ATB2Bin2;[STAT][LOG][SNMP:#:1:30];ATB - Printer Stock 2 available
@@ -1517,6 +1523,10 @@
 100722;massai.notification.drivers.Sel.34;[STAT][LOG][SNMP:#:1:30];Special Electronic ALERT_CODE_TEMPERATURE_OK
 100723;massai.notification.drivers.Sel.35;[STAT][LOG][SNMP:#:1:30];Special Electronic ALERT_CODE_TEMPERATURE_HIGH
 
+101000;massai.notification.drivers.ScaleDev.Scale.0;[STAT][LOG][SNMP:#:1:30];Scale available
+101002;massai.notification.drivers.ScaleDev.Scale.2;[STAT][LOG][SNMP:#:1:30];Scale hard error
+101003;massai.notification.drivers.ScaleDev.Scale.3;[STAT][LOG][SNMP:#:1:30];Scale disconnected
+
 
 108000;massai.notification.drivers.GPPDev.1;[STAT][LOG][SNMP:#:1:30];General-purpose-printer available
 108001;massai.notification.drivers.GPPDev.2;[STAT][LOG][SNMP:#:1:30];General-purpose-printer disconnected
diff --git a/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h b/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
index 7708d9f0..0713cc9c 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
+++ b/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
@@ -11,8 +11,8 @@
 
 #include "Tools/VERSION.H"
 
-#define FILE_VER_BIN     1,0,4,3
-#define FILE_VER_STR     "FileVersion", "1.0.4.3\0"
+#define FILE_VER_BIN     1,0,4,4
+#define FILE_VER_STR     "FileVersion", "1.0.4.4\0"
 
 /*!
 \page LogService.exe - Logging Service
@@ -22,10 +22,14 @@
 
 !!! immer auch wenn nötig massaiSNMP_def.h pflegen !!!
 
-@version 1.0.4.3
+@version 1.0.4.4
 
 \section change1 Change history
 
+\date 31.01.2013 \li V1.0.4.4
+                 \li SteT Mantis 0025566: WeightScale: support standalone Scale (BaggageScale CUSS1.3)
+                 \li new event-/comp-configuration for WeightScaleDevice and massai.notification.drivers.ScaleDev.Scale
+
 \date 22.01.2013 \li V1.0.4.3
                  \li SteT Mantis 0025263: DeskoPenta
 
diff --git a/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h b/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
index 7b8348fb..575f8ab7 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
+++ b/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
@@ -11,8 +11,8 @@
 
 #include "Tools/VERSION.H"
 
-#define FILE_VER_BIN     1,0,4,3
-#define FILE_VER_STR     "FileVersion", "1.0.4.3\0"
+#define FILE_VER_BIN     1,0,4,4
+#define FILE_VER_STR     "FileVersion", "1.0.4.4\0"
 
 /*!
 \page massaiSNMP.dll - plugin for MS SNMP-Service;
@@ -22,10 +22,14 @@
 
 !!! immer auch wenn nötig logService_def.h pflegen !!!
 
-@version 1.0.4.3
+@version 1.0.4.4
 
 \section change1 Change history
 
+\date 31.01.2013 \li V1.0.4.4
+                 \li SteT Mantis 0025566: WeightScale: support standalone Scale (BaggageScale CUSS1.3)
+                 \li new event-/comp-configuration for WeightScaleDevice and massai.notification.drivers.ScaleDev.Scale
+
 \date 22.01.2013 \li V1.0.4.3
                  \li SteT Mantis 0025263: DeskoPenta
 
-- 
2.41.0.windows.1

